Gruntfile.js

```diff
@@ -128,4 +180,13 @@ module.exports = function (grunt) {
         grunt.task.run("shell:runUnitTests");
         grunt.task.run("shell:runAcceptanceTests");
     });
+    grunt.registerTask("buildRpmDocker", "Build Linux RPM package using a Docker container", function () {
+        grunt.task.run("shell:buildRpmDocker");
+    });
+    grunt.registerTask("buildrpm", "Build GPII Linux and RPM package", function () {
```

Comment:
I would suggest hyphenating the task name to read ``build-rpm``. That naming scheme seems to be used for other GPII Grunt tasks https://github.com/GPII/grunt-gpii/blob/master/tasks/gpii.js